### ISSUE TITLE
Global NavBar - Desktop, Tablet

### ISSUE THREAD

**Issue by randelbrot** [hackforla/HomeUniteUs#493]
#### The Problem:

Home Unite Us users (Coordinators, Guests, and Hosts) do not currently have a clear and simple way to navigate to global (i.e., not user specific or section specific) functionality after logging into the Home Unite Us application.

As a result, users cannot access critical functionality within the application, including Settings, Help, and Log Out.

#### High Level Approach:

Provide a simple navigation bar for all users types that links to critical functionality only (Settings, Help, Log Out). This phase will focus on desktop and tablet only, a separate issue will cover mobile.

#### Considerations:

- This phase will only include links for critical functionality, additional functionality will be added in the future (e.g., notifications).
- This phase will include the Home Unite Us logo, future phases may support custom organization logos

#### Goals & Success:

Success is if each user type can easily navigate to critical functionality on desktop and tablet.

#### Key Features:

- Navigation bar
   - Left-aligned: Home Unite Us logo
   - Right-aligned: Avatar with the user's first and last initial (no photo)

- Dropdown, when the avatar is clicked
   -Settings: when clicked, redirect users to a placeholder Settings page
   -Help: when clicked, open in a new tab: https://github.com/hackforla/HomeUniteUs/wiki/Home-Unite-Us-User-Guide 
   -Log Out: when clicked, redirect users to the Home Unite Us home page (homeunite.us)

#### Design:

- [x] #493
- [x] Edit existing designs as needed
- [x] Designs are edited and ready for engineering. 

#### Branding:

Use the Home Unite Us logo

#### Known Limitations:

- This issue doesn't include mobile, a separate issue will include mobile

#### Team Reviews:

- [x] Product - Bryce
- [x] Design - Ed
- [x] Engineering - Erik

**Comment by edela0015** [comment]
Design is currently working on this. A  consistent nav bar with tailored functionality.

**Comment by edela0015** [comment]
Proposed but not ready for dev: 

https://www.figma.com/file/BNWqZk8SHKbtN1nw8BB7VM/HUU-Everything-Figma-Nov-2022?type=design&node-id=7858-105237&mode=design&t=4ULAdJTdYAui4lvV-0

**Comment by brycelednar** [comment]
Hey @edela0015, thanks for sharing the Figma!

The Figma that you created actually includes more functionality than is needed for this issue. This issue focuses on getting the logo and avatar in the correct position and creating the dropdown list from the avatar. 

Here is a rough wireframe I created as a starting point for this one: https://www.figma.com/file/BNWqZk8SHKbtN1nw8BB7VM/HUU-Everything-Figma-Nov-2022?type=design&node-id=6734-93274&mode=design&t=56mhnOy3fBIS6QFp-4 . 

For other functionality like the links to the Dashboard and My Documents, that functionality would be covered by a separate, future issue.

**Comment by edela0015** [comment]
Thank you, Bryce. I updated the figma. Please let me know if there are more questions. the logo is embedded into the side nav. we decided this on the last design meeting so let me know if this direction can be "okayed."

**Comment by brycelednar** [comment]
Thanks for those updates, @edela0015. I left a [comment](https://www.figma.com/file/BNWqZk8SHKbtN1nw8BB7VM?node-id=7858:105237&mode=design#485369601) in the Figma for you. Let me know if I can clarify anything.

**Comment by erikguntner** [comment]
@brycelednar Let me know if this meets the requirements.

https://github.com/hackforla/HomeUniteUs/pull/536

**Comment by brycelednar** [comment]
@erikguntner just to confirm, is this compatible with tablet as well? If so, this meets requirements - thanks! Once the review is done on the PR I will close this issue.

Hi @edela0015, this issue is all set from a design perspective, thanks!

**Comment by erikguntner** [comment]
@brycelednar If it's supposed to look and function the same on tablet as it does on desktop, then yes, this is compatible.

**Comment by brycelednar** [comment]
Roger that, thanks @erikguntner !

**Comment by pixellady** [comment]
Hi @lola3736 
I have completed the sanity check on this Issue 493. No issues noted, good to pass on to Design to create Wireframes.

**Comment by pixellady** [comment]
Reopened it to perform sanity check as a part of account creation.
